ta/crypt/derive_key_taf.c

```diff
@@ -0,0 +1,220 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2019, Linaro Limited
+ */
+
+#include <pta_system.h>
+#include <string.h>
+#include <tee_internal_api.h>
+
+#include "derive_key_taf.h"
+
+#define TA_DERIVED_KEY_MIN_SIZE 16
```

Comment:
we lost tabulations here...

Reply:
Never added, I did that in optee_os ... will do it now.